packages/react/src/components/header/components/navigationLink/NavigationLink.module.scss

```diff
@@ -1,32 +1,78 @@
+@use "../../Header.module.scss" as header;
+@import "../../../../styles/common.scss";
+
 .navigationLink.navigationLink {
-  --link-visited-color: none;
   --link-color: var(--color-black-90);
+  --link-font-size: 14px;
+  --link-visited-color: none;
+
+  display: flex;
+  flex-grow: 1;
+  line-height: var(--lineheight-l);
+  margin: 0 calc(-1 * var(--header-focus-outline-width));
+  padding: 13px 0 15px;
+  text-decoration: none;
+  word-wrap: break-word;
+
+  &:hover,
+  &:focus {
```

Comment:
Fix focus area everywhere (universal bar, action bar links, navigation links on desktop and mobile. Consult [designs](https://app.abstract.com/projects/bf7025a0-5b50-11e9-80c2-5323106dd058/branches/958cf4cb-b29b-4eab-aa1c-0a7a892ce6e1/commits/latest/files/759b250d-52d4-4728-ab73-cfe0436f060c/layers/E39E6420-0E6C-4452-AC4E-BF76E36F5F7A?collectionId=e07fc52a-9dee-4617-946f-5dd1ee787196&collectionLayerId=6a182692-3841-40b7-a80c-57d427b8b352) and [hel.fi](https://www.hel.fi/fi)

Reply:
This issue still persists in universal and navigation bar's links (only on top level, nested links are fine), as the focus areas are much higher than in the designs.